src/wallet/mod.rs

```diff
@@ -485,9 +493,9 @@ where
         // the new tx must "pay for its bandwidth"
         let vbytes = tx.get_weight() as f32 / 4.0;
         let required_feerate = FeeRate::from_sat_per_vb(details.fees as f32 / vbytes + 1.0);
-        let new_feerate = builder.fee_rate.unwrap_or_default();
+        let new_feerate = get_fee_rate(&builder.fee_policy);
 
-        if new_feerate < required_feerate {
+        if new_feerate < required_feerate && !builder.has_absolute_fee() {
```

Comment:
This doesn't check whether the new absolute fee is actually high enough to "pay for its bandwidth".